src/app/shared-services/db/migrations.ts

```diff
@@ -6,19 +6,18 @@ export async function runMigrations(
 ) {
   const init = `CREATE TABLE IF NOT EXISTS migrations (
     version integer NOT NULL PRIMARY KEY AUTOINCREMENT,
-    up TEXT NOT NULL,
-    down TEXT);`
+    up TEXT NOT NULL);`
   const {
     changes: { changes },
     message,
   } = await db.execute({ statements: init })
   if (changes === -1) throw new Error(`can't run DB migrations: ${message}`)
 
   const { values } = await db.query({
-    statement: `SELECT count() FROM migrations;`,
+    statement: `SELECT version FROM migrations ORDER BY version DESC LIMIT 1;`,
     values: [],
   })
-  const currentVersion = parseInt(values[0]['count()'], 10)
+  const currentVersion = parseInt(values[0].version, 10)
```

Comment:
This approach does not seem to work in iOS: `values` is an empty array. The result of the table migrations table (`changes`) is 0. Maybe that's related?

Reply:
Damn,the new query didn't catch the case of `no migrations ran yet`, it should work now.

Reply:
Check, it works!